frontend/src/App.vue

```diff
@@ -1,32 +1,23 @@
 <script setup lang="ts">
 import { RouterLink, RouterView } from 'vue-router'
-import HelloWorld from './components/HelloWorld.vue'
+import MainHeader from './components/Header/MainHeader.vue';
+import MainFooter from './components/Footer/MainFooter.vue';
 </script>
 
 <template>
   <header>
-    <img alt="Vue logo" class="logo" src="@/assets/logo.svg" width="125" height="125" />
-
-    <div class="wrapper">
-      <HelloWorld msg="You did it!" />
-
-      <nav>
-        <RouterLink to="/">Home</RouterLink>
-        <RouterLink to="/about">About</RouterLink>
-      </nav>
-    </div>
+    <MainHeader />
   </header>
-
   <RouterView />
+  <footer>
+    <MainFooter/>
+  </footer>
 </template>
 
 <style scoped>
-header {
-  line-height: 1.5;
-  max-height: 100vh;
-}
 
-.logo {
+
+/* .logo {
```

Comment:
I recommend you to remove commented code, if you'll need it you can check in git history